cli/cmd/usergroup/list_user.go

```diff
@@ -0,0 +1,41 @@
+package usergroup
+
+import (
+	"github.com/rilldata/rill/cli/pkg/cmdutil"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	"github.com/spf13/cobra"
+)
+
+func ListUserCmd(ch *cmdutil.Helper) *cobra.Command {
+	var group string
+
+	addCmd := &cobra.Command{
+		Use:   "list-user <name>",
```

Comment:
Similar comment – torn on this, but if we want just one way of doing things, this might better fit in `rill user list --group NAME`, similar to `rill user list --project NAME`.